### ISSUE TITLE
Spans not displayed in UI using docker on windows 10 hyper-v VM

### ISSUE THREAD

**Issue by stephen00001** [jaegertracing/jaeger#2286]
**Describe the bug**
I run the Hotrod example on a windows 10 hyper-v vm.
I press the buttons on the app and when I open the Jaeger UI there are no traces displaying.
The metrics page reports the spans are received. 
Also the services appear in the dropdown list of the UI

**To Reproduce**
1. Launch a windows 10 hyper-v instance 
2. Run [Hotrod ](https://github.com/jaegertracing/jaeger/tree/master/examples/hotrod) . add admin port (-p14269:14269)
3. Open browser and press some of the buttons on the Hotrod app
4. Open Jaeger UI and see no spans are displayed 

**Expected behavior**
Spans should be displayed in the UI

**Screenshots**
![image](https://user-images.githubusercontent.com/66700665/84202125-18748480-aaa0-11ea-8a88-6c092f108cbf.png)

**Version (please complete the following information):**
 - OS: Windows 10 on hyper-v
 - Jaeger version: latest, assuming that is 1.18.0
 - Deployment: docker for windows on windows 10 hyper-v

**What troubleshooting steps did you try?**
Try to follow https://www.jaegertracing.io/docs/latest/troubleshooting/ and describe how far you were able to progress and/or which steps did not work.
I found the issue on another application I am building and did all of the above on that application with exact same results. I've attached the metrics from the Hotrod example.
I also tried Hotrod example on a baremetal windows 10 and it works fine
[metrics.txt](https://github.com/jaegertracing/jaeger/files/4754890/metrics.txt)

**Additional context**
I've more details on [stackoverflow ](https://stackoverflow.com/questions/62273414/issue-sending-traces-from-a-docker-container-to-a-jaeger-docker-container-runnin) regarding the issue occurring with my original app.

**Comment by yurishkuro** [comment]
I wonder if you have clock issue where the timestamps reported in the spans are different from what UI thinks the current time is, and therefore the time window that UI sends to the server does not match the span data.

**Comment by stephen00001** [comment]
You were spot on. Although the host machine (VM) updated its clock on every unpause, docker for windows did not. The timezones were correct for all containers but the times were all out by the exact same amount. This must be the docker internal clock that seems to only get updated once on launch and not at the launch of every new container. Although all container clocks were out by the same amount, the windows host machine was correct. I'm assuming the timestamps on the network packets got (re)stamped with the host machine time  when they went via the host network adapter. 
Thanks for the suggestion as I never would have thought of looking at the clocks.

Is there a way to turn off the timestamp checking in Jaeger as I'm sure the spans contain all the relevant information that needs to be logged anyway?

**Comment by yurishkuro** [comment]
it's not "checking" per se, the time range is a filtering parameter required by the storage implementations. The UI defaults the time range to `[now-lookback, now]`, but if your `now` is far in the past, then it won't find any traces.

**Comment by stephen00001** [comment]
Thanks. I understand now what was happening. I wrote the last message very late last night and was clearly not thinking straight. It's as simple as the messages were arriving but the times/dates were outside the time frame window the UI was displaying. If I set a custom date range I'm sure they would appear. The containers must have been out by a few days with the continual stopping and starting and not by just a few hours.
